eric.txt

```diff
@@ -0,0 +1 @@
+My name is Eric, my favorite robot is rover
```

Comment:
Line 1: `My name is Eric, my favorite robot is rover` should be `My name is Eric, my favorite robot is auv`